Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Distributions"
 uuid = "31c24e10-a181-5473-b8eb-7969acd0382f"
 authors = ["JuliaStats"]
-version = "0.24.17"
+version = "0.24.18"
```

Comment:
What is the convention in Distributions - is the different type parameter of `MixtureModel` considered a breaking change?

Reply:
Yes

Reply:
OK, I updated the version number.

Reply:
But now tests fail since there exists no version of HypothesisTests that is compatible with Distributions 0.25 🤔

Reply:
I opened a PR that removes the test dependency on HypothesisTests to fix these test failures: https://github.com/JuliaStats/Distributions.jl/pull/1311